docs/tutorials/99_other/publish.md

```diff
@@ -0,0 +1,54 @@
+# Tutorial - How to Publish data and metadata on the OEP
+
+<!-- keep img below title and without align="left"  -->
+<img src="https://raw.githubusercontent.com/OpenEnergyPlatform/academy/develop/docs/data/img/OEP_logo_2_no_text.svg" alt="OpenEnergy Platform" height="75" width="75" />
+
+## Learnings
+
+This tutorials will enable you to:
+
+- Publish your data and metadata 
+	- with an embargo
+	- to a topic
+
+## Requirements
+
+- OEP Account
+- a table or dataset with metadata uploaded to the [OpenEnergyPlatform](https://openenergyplatform.org/) (OEP)
+
+
+## Introduction
+
+This course will teach you step-by-step how to publish your data and metadata. Comment: Later will be a corresponding course linked here.
+
+
+## Choosing a table to be published
+If you want to publish a table (including [data and] metadata) you have to go to your [profile page](https://openenergyplatform.org/user/profile/959/settings) on the OEP.
+Now select the tab [`Tables`](https://openenergyplatform.org/user/profile/959/tables). Now you see all tables you contributed; First are listed all tables you have already published, if you have not published any tables yet, this field will show 'You have never published a table.'.
+Beneath are listed all of your tables in the [model draft](https://openenergyplatform.org/dataedit/view/model_draft) section. 
+From this tables you can select the table you want to publish. Note: Only tables with an valid id and license in their metadata can be published on the OEP. Stimmt das so?
+
+## Choosing an embargo and a topic
+Once you selected the table you want to publish, a pop-up field `Publish Table` will open. Next to the fields is an `i` icon giving more information if you hover your mouse over it.
+First you can apply an embargo; in a drop down menu you can select between none, and a 6 months or 1 year embargo period.
```

Comment:
‘none’ also removes a embargo period if it is present. This may be the case if it was set when the data was uploaded.